### DISCUSSION TITLE
A vscode like gruvbox?

### DISCUSSION THREAD

**Discussion by rew1nter** [ellisonleao/gruvbox.nvim#357] [General]
Hey there, I was wondering if it's possible to make the colorscheme look like vscode gruvbox

**Comment by rew1nter** [comment]
Here's a photo for reference
![image](https://github.com/user-attachments/assets/56c3533d-0941-4a4b-a7e5-3d53149d1ff6)

**Reply by polyzen** [reply]
Perhaps what you're seeing is [Semantic Tokens](https://github.com/golang/tools/blob/master/gopls/doc/features/passive.md#semantic-tokens)? Seems like it would be possible to use gopls's Semantic Tokens support in Neovim, but don't know how involved that would be. 
Here's a PR aiming to add it to Eglot (Emacs LSP client): https://github.com/joaotavora/eglot/pull/839

**Comment by ghost** [comment]
hey there, could be wrong but doesn't it explain overriding colours in the readme?

**Reply by rew1nter** [reply]
yes but I don't know exactly which colors to override

**Reply by ghost** [reply]
> yes but I don't know exactly which colors to override

Trial and error! :D

**Reply by ellisonleao** [reply]
hey @sealpup321 you can use the `:Inspect` command with the current cursor in the text you want to checkout the color and group

**Comment by smsearcy** [comment]
@rew1nter did you figure out what to override?  Out of the box I think this colorscheme has too much green, so I'd like to do some similar overrides for the function names and if you had something to copy that would be great!

**Reply by rew1nter** [reply]
Nah man. I just gave up on the whole thing and switched to sonokai :(
